en-US/S.xml

```diff
@@ -112,7 +112,7 @@
         </table>
       </listitem>
     </varlistentry>
-		 <varlistentry id="see">
+    <!-- 		 <varlistentry id="see">
```

Comment:
Is the "see" entry being removed from the word usage guide? I suggest keeping it, saying something like "not preferred" or "do not use",  and providing some guidance about using more accessible language, possibly with examples. "See" comes up in a few contexts for content -- not only for references, but also when describing screenshots or output.

Reply:
Here's a suggested entry for "see". I'm unsure about the examples -- input welcome! 

For references within a document, and for cross-references to other resources, do not use “see”. Use “refer to”.

For descriptions of graphics, explanations of command outputs, and other similar cases, it is acceptable to use “see”. At the same time, it might be more concise or precise to replace “see” with another verb in these cases. 

Examples: 
Original: Click a server link to see the result details:
Update: Click a server link for the result details:
Update: For the result details, click a server link: 

Original: As you can see, these are expected Audit events with an `AVC` Audit record.
Update: As shown in the output, these are expected Audit events with an `AVC` Audit record.

Reply:
Good suggestion - thanks!